### PR TITLE
fix(toolbar-padding): updated toolbar padding

### LINKED ISSUE #3240: Toolbar padding enhancements
- ~~Update toolbar padding to increase from `--md` to `--lg` at medium breakpoint, like `page__main-section`, `card`, etc~~
- Update toolbar padding values for `.pf-c-page__main-section`, `.pf-c-card`, `.pf-c-table` etc to contextually reflect padding left/right 
- Add `pf-m-no-padding` variant.

### PR DESCRIPTION
**Pull request by mattnolting**
fixes #3240

Initial PR here: https://github.com/patternfly/patternfly/pull/3317

### CODE REVIEW

**Comment by mcoker** [comment]
I was talking with @mcarrano about this and we think it's too risky to change the default padding as we don't want to introduce a visual bug for users using the toolbar currently.

We can offer the inset system as a way to get the padding, but wdyt about adding a variation that adds this padding, then updating the default padding later in a breaking change release?

File: src/patternfly/components/Toolbar/examples/Toolbar.md [hunk 1]
```diff
@@ -151,12 +151,64 @@ In some instances, it may be necessary to adjust spacing explicitly where items
...
+    {{/toolbar-content-section}}
+  {{/toolbar-content}}
+{{/toolbar}}
+```
+
+### Toolbar spacers and insets
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
+| `.pf-m-page-insets` | `.pf-c-toolbar` | Modifies toolbar insets to match page section, table, page header or any other component whose inset shifts from `--pf-global--spacer--md` to `--pf-global--spacer--lg` at the `xl` breakpoint. |
```
Comment (mcoker) [inline src/patternfly/components/Toolbar/examples/Toolbar.md:208]: I imagine it's a good idea to reuse this modifier for any other component that needs this kind of variant for spacing. @christiemolloy wdyt about the name?

Comment (christiemolloy) [reply src/patternfly/components/Toolbar/examples/Toolbar.md:208]: yeah i think the naming works!

**Pull request merged**